src/System.Diagnostics.TextWriterTraceListener/tests/DelimiterWriteMethodTests.cs

```diff
@@ -8,14 +8,14 @@
 
 namespace System.Diagnostics.TextWriterTraceListenerTests
 {
-    public class DelimiterWriteMethodTests : IDisposable
+    public class DelimiterWriteMethodTests : FileCleanupTestBase
     {
         private readonly Stream _stream;
         private readonly string _fileName;
 
         public DelimiterWriteMethodTests()
         {
-            _fileName = string.Format("{0}.xml", GetType().Name);
+            _fileName = $"{GetTestFilePath(null, GetType().Name)}.xml";
```

Comment:
Yes I meant that, since we are calling it from the constructor, [CallerMemberName] would be the same as the test class since the constructor is called the same. Thanks for the heads up